lib/active_model/serializer/railtie.rb

```diff
@@ -1,9 +1,11 @@
+require 'active_model_serializers'
 require 'rails/railtie'
+
 module ActiveModel
   class Railtie < Rails::Railtie
     initializer 'active_model_serializers.logger' do
-      ActiveSupport.on_load(:action_controller) do
-        ActiveModelSerializers.logger = ActionController::Base.logger
+      ActiveSupport.on_load(:active_model_serializers) do
+        self.logger = ActionController::Base.logger
       end
```

Comment:
bonus code, out of scope... but shouldn't be that controversial.. pattern copied from active job